__main__.py

```diff
@@ -11,27 +11,45 @@
 from email.utils import make_msgid
 import optparse
 
-client_id_path = "./atefics/client.json"
+client_id_path = "/reg/g/pcds/epics-dev/nrw/ATEF/atefics/client.json"
 
 """Performs an ATEF check given a config file and sends a corresponding
 email to a list of recepients. 
 
 """
+devices = ['SP1K1 MONO', 'MR1K1 BEND', 'MR2K2 FLAT', 'MR3K2 KBH']
+
+def FilePathUtil(config_file):
+    if os.path.isfile(config_file):
+        return config_file
+    elif os.path.isfile("./atefics/checkouts/" + config_file):
+        return "./atefics/checkouts/" + config_file
+    else:
+        print(f"{config_file} does not exist")
+        sys.exit(-1)
```

Comment:
The more "pythonic" thing to do is raise an exception (maybe `FileNotFoundError`), then either let that stop your program or catch it higher up and return.  But nothing strictly wrong with this

Reply:
updated.